### ISSUE TITLE
why do the tabs look awful -- the text is gobbledygook

### ISSUE THREAD

**Issue by yakshaving** [warpdotdev/Warp#5605]
### Dupe Check

- [X] I have [searched Warp bugs](https://github.com/warpdotdev/warp/issues?q=is%3Aissue+label%3ABugs) and there are no duplicates

### Describe the bug

<img width="435" alt="image" src="https://github.com/user-attachments/assets/336f0dd3-c45a-4e57-89c4-7e96387b0948" />

### To reproduce

Just open and use warp.. 

### Operating system

MacOS

### Operating system and version

latest

### Regression

No, this bug or issue has existed throughout my experience using Warp

### Does this block you from using Warp daily?

No

### Is this an issue only in Warp?

Yes, I confirmed that this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b9d78064-c89e-4973-b153-5178a31ee54e

None

**Comment by heysweet** [comment]
Hello! Do you have any other details about possible reproduction here? I'm not seeing this on my Mac. Is this for all tabs on your machine? Not only does it look weirdly formatted but the contents look interesting to me. Any idea what the characters are that we're seeing?

**Comment by yakshaving** [comment]
Not really, I've never seen anything like this on any other app.  Tried
changing editor font but that's obviously different.  Didn't see anything
else in settings thst could manipulate this.  🤷🏽‍♂️

**Comment by heysweet** [comment]
I was at a bit of a loss, so I had a teammate take a quick look and this is what they said:

"They might've accidentally catted some binary file that happened to have the escape sequences to set a custom tab name. If so, they can right-click on the tab and reset its name. That option only appears if the name is custom, so if it's not there something else is going on."

**Comment by yakshaving** [comment]
Are you sure?   I don’t think I did that, but then, wouldn’t it just start working automatically then the next time when I open up Warp?   It does this every single time.

**Comment by heysweet** [comment]
<img width="282" alt="image" src="https://github.com/user-attachments/assets/481438fc-1b0b-40a3-8cb4-7521e35c0060" />

Do you see the "Reset tab name" option? This also could help diagnose!

**Comment by yakshaving** [comment]
Somehow reset tab name disappeared for me - I saw it once but I don’t see it anymore




When I hit Rename:



After I hit enter…





----

**Comment by heysweet** [comment]
> Somehow reset tab name disappeared for me - I saw it once but I don’t see it anymore When I hit Rename: After I hit enter…
> […](#)
> ----
> On Dec 20, 2024 at 8:15 AM -0800, Andrew Sweet ***@***.***>, wrote: image.png (view on web) Do you see the "Reset tab name" option? This also could help diagnose! — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***>

Unfortunately it looks like replying via email is not including any images if you’re attaching images

**Comment by yakshaving** [comment]
I think I might see the issue here - Does this not work with ohmyzsh?  https://ohmyz.sh/. Whenever this gets sourced it seems to mess up.
